tests/hpc/mrsh_slave.pm

```diff
@@ -13,7 +13,7 @@
 # Maintainer: soulofdestiny <mgriessmeier@suse.com>
 # Tags: https://fate.suse.com/321722
 
-use base "hpcbase";
+use base "opensusebasetest";
```

Comment:
There's a `$self->enable_and_start` in line 39.

Reply:
Changed!